src/components/unsigned-sweep.js

```diff
@@ -176,15 +180,17 @@ class UnsignedSweep extends Component {
       };
 
       // Retrieve the desired file path and file name
-      const filePath = dialog.showSaveDialog(dialogParams);
+      const filePath = await dialog.showSaveDialog(dialogParams);
       if (!filePath) {
         // TODO: The user exited the file creation process. What do we do?
         return;
       }
 
-      fs.writeFileSync(filePath, JSON.stringify(recoveryPrebuild, null, 4), 'utf8');
-      this.setState({ recovering: false, done: true, finalFilename: filePath });
+      fs.writeFileSync(filePath.filePath, JSON.stringify(recoveryPrebuild, null, 4), 'utf8');
+      this.setState({ recovering: false, done: true, finalFilename: filePath.filePath });
+      alert('We recommend that you use a third-party API to decode your txHex and verify its accuracy before broadcasting.');
     } catch (e) {
+      console.dir(e);
```

Comment:
This logs the error to the console if there are any errors (which we would ask the customers to take a screenshot for)... otherwise it would be kinda hard to see the stack traces when debugging. I see this `console.dir` as the easiest way to add logging with our time constraint.